### PR TITLE
Added additional Loki Operator installation options

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves [OCPQE-13548](https://issues.redhat.com/browse/OCPQE-13548)

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Successfully tested on a Flexy `ci` cluster:
Unreleased (Loki 5.6): https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/323/
Released (failed b/c Loki 5.5 is not supported in CI at this time): https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/322/

**Comment by nathan-weinberg** [comment]
Successfully tested on a Flexy non-`ci` cluster:
Unreleased (Loki 5.6): https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/332/
Released (failed b/c Loki 5.5 is not supported in CI at this time): https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/330/

**Review by memodi — APPROVED** [review]
/lgtm

File: scripts/subscriptions/loki-unreleased-subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: loki-operator
+  namespace: openshift-operators-redhat
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: loki-operator
+  source: qe-app-registry
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/subscriptions/loki-unreleased-subscription.yaml]: that's an old changeset

**Pull request merged**